js/templates/modals/editListing/uploadPhoto.html

```diff
@@ -1,6 +1,6 @@
 <li style="background-image: url('<%= ob.getServerUrl(`ob/images/${ob.small}`) %>')" class="tile">
   <div class="primaryIndicatorWrap">
-    <div class="clrE1 clrTOnEmph phraseBox padTn">PRIMARY PHOTO</div>
+    <div class="clrE1 clrTOnEmph phraseBox padTn"><%= ob.polyT('primaryPhoto').toUpperCase() %></div>
```

Comment:
I think we should leave the uppercase in the translation instead of forcing it here. Uppercase letters may mean something different in other languages.